ircd/channel.c

```diff
@@ -1966,8 +1973,8 @@ modebuf_mode(struct ModeBuf *mbuf, unsigned int mode)
 
   mode &= (MODE_ADD | MODE_DEL | MODE_PRIVATE | MODE_SECRET | MODE_MODERATED |
 	   MODE_TOPICLIMIT | MODE_INVITEONLY | MODE_NOPRIVMSGS | MODE_REGONLY |
-           MODE_NOCOLOR | MODE_NOCTCP |
-           MODE_DELJOINS | MODE_WASDELJOINS | MODE_REGISTERED);
+           MODE_NOCOLOR | MODE_NOCTCP | MODE_MODERATENOREG | MODE_DELJOINS |
```

Comment:
This should not move MODE_DELJOINS.
```suggestion
           MODE_DELJOINS | MODE_NOCOLOR | MODE_NOCTCP | MODE_MODERATENOREG |
```